SPECS-EXTENDED/SDL/SDL.spec

```
@@ -0,0 +1,820 @@
+Summary:        A cross-platform multimedia library
+Name:           SDL
+Version:        1.2.15
+Release:        45%{?dist}
+# The license of the file src/video/fbcon/riva_mmio.h is bad, but the contents
+# of the file has been relicensed to MIT in 2008 by Nvidia for the
+# xf86_video-nv driver, therefore it can be considered ok.
+# The license in the file src/stdlib/SDL_qsort.c is bad, but author relicensed
+# it to zlib on 2016-02-21,
+# <https://www.mccaughan.org.uk/software/qsort.c-1.14>, bug #1381888.
+License:        LGPLv2+
```

```suggestion
License:        LGPL-2.1-or-later
```
Reference: https://github.com/microsoft/CBL-Mariner/pull/4506#discussion_r1058527431

---

Done, update the license string.